src/precice/tests/configuration.xml

```diff
@@ -36,8 +36,8 @@
          <read-data  name="Displacements"  mesh="PeanoNodes" />
          <read-data  name="Velocities"     mesh="PeanoNodes" />
          <read-data  name="VelocityDeltas" mesh="PeanoNodes" />
-         <export:vtk timestep-interval="100" directory="output/"/>
-         <export:vtk timestep-interval="100" directory=""/>
+         <export:vtk time-window-interval="100" directory="output/"/>
+         <export:vtk time-window-interval="100" directory=""/>
```

Comment:
Here, I again do not really like both names.

Reply:
If we clearly state that export only takes place at the end of a time window, we can simply call this `output-frequency`/`export-frequency` -- even shorter `frequency` , if we are inside the `<export:vtk ... />` element, since it is clear, that we are dealing with output/export.